src/bun.js/bindings/headers-handwritten.h

```diff
@@ -106,7 +106,7 @@ typedef struct ResolvedSource {
     BunString source_code;
     BunString source_url;
     bool isCommonJSModule;
-    uint32_t hash;
```

Comment:
This change needs to be reflected in `javascript.zig` as well

Reply:
???

Reply:
ResolvedSource in our Zig code still has `hash` (`bindings/ResolvedSource.zig`)